test/jdk/com/sun/jdi/TestScaffold.java

```diff
@@ -453,6 +453,9 @@ protected void traceln(String str) {
 
     protected void failure(String str) {
         println(str);
+        StackTraceElement[] trace = Thread.currentThread().getStackTrace();
+        for (StackTraceElement traceElement : trace)
```

Comment:
Please add curly braces around the cycle body

Reply:
Ok. This was a copy-n-paste from Throwable.printStackTrace(), and I didn't noticed the omission when copying.